lms/djangoapps/teams/tests/test_views.py

```diff
@@ -1397,3 +1400,55 @@ def test_bad_username(self):
 
     def test_missing_membership(self):
         self.delete_membership(self.wind_team.team_id, self.users['student_enrolled'].username, 404)
+
+
+class TestElasticSearchErrors(TeamAPITestCase):
+    """Test that the Team API is robust to Elasticsearch connection errors."""
+
+    ES_ERROR = ConnectionError('N/A', 'connection error', {})
+
+    def setUp(self):
+        super(TestElasticSearchErrors, self).setUp()
+        CourseTeamIndexer.engine().destroy()
+        for team in self.test_team_name_id_map.values():
```

Comment:
I had lifted those lines from @bderusha's tests, but it looks like they aren't actually necessary for my purposes. They've been removed.